docs/user_guide/backend_configuration.rst

```diff
@@ -133,10 +134,10 @@ Then we can use this configuration to write the NWB file:
 
     from neuroconv.tools.nwb_helpers import configure_backend, BACKEND_NWB_IO
 
-    dataset_configurations["acquisition/MyTimeSeries/data"] = dataset_configuration
+    backend_configuration.dataset_configurations["acquisition/MyTimeSeries/data"] = dataset_configuration
```

Comment:
This is to keep the horizontal width from getting too large in the central pydata window

Reply:
Does it actually work if you don't assign it back to the `backend_configuration` object though?

Reply:
Yes, because the `dataset_configurations` variable is a pointer to that part of the `backend_configuration`, not a copy

Reply:
In particular, this is what happens without breaking it up into separate lines

![image](https://github.com/catalystneuro/neuroconv/assets/51133164/da354694-3972-483c-b5a5-8042c0dea26c)

Though it also increases readability in general IMO

Reply:
OK I have confirmed that this is true, but then I think it would be clearer if we printed out `print(backend_configuration)` to demonstrate this

Reply:
But then you show the unneeded information about the unmodified `timestamps` field

I don't think most people will be confused by this minor detail

Reply:
I think it might, but I'm OK with keeping it as is for now